app/models/plant_population.rb

```diff
@@ -21,28 +21,39 @@ class PlantPopulation < ActiveRecord::Base
                           foreign_key: 'plant_population_id',
                           association_foreign_key: 'plant_line_name'
 
-  # Exlude the ['none', 'unspecified', 'not applicable'] pseudo-record trio
-  scope :drop_dummies, -> do
-    where.not(canonical_population_name: '')
+  include Filterable
+
+  scope :by_name, -> { order(:plant_population_id) }
+
+  def self.grouped(params = nil)
+    count = 'count(plant_lines.plant_line_name)'
+    query = (params && params[:query].present?) ? filter(params) : all
+    query.
+      select(table_columns + [count]).
```

Comment:
Yes, I discovered that also. But after some fiddling with the code I decided to revert most of that I forgot about that. Will fix.